addons/common/XEH_postInit.sqf

```diff
@@ -176,6 +176,12 @@
 [QGVAR(selectWeapon), {
     params ["_unit", "_muzzle"];
     _unit selectWeapon _muzzle;
+    if (_muzzle == secondaryWeapon _unit) then {
+        _unit playMove "amovpercmstpsraswlnrdnon";
+        _unit disableAI "ANIM";
+    } else {
+        _unit enableAI "ANIM";
+    };
```

Comment:
I think this is a decent workaround. However, IMO would be better placed in the `switchWeapon` function itself (since it is specific to that).

Reply:
switchWeapon function is local to the curator, and these need to be local to the AI. should I make a separate selectWeaponCurator event?